packages/graphql/tests/schema/directives/relationship-aggregate.test.ts

```diff
@@ -0,0 +1,2953 @@
+                  pageInfo: PageInfo!
+                  totalCount: Int!
+                }
+
+                type Mutation {
+                  createActors(input: [ActorCreateInput!]!): CreateActorsMutationResponse!
+                  createMovies(input: [MovieCreateInput!]!): CreateMoviesMutationResponse!
+                  deleteActors(where: ActorWhere): DeleteInfo!
+                  deleteMovies(delete: MovieDeleteInput, where: MovieWhere): DeleteInfo!
+                  updateActors(update: ActorUpdateInput, where: ActorWhere): UpdateActorsMutationResponse!
+                  updateMovies(connect: MovieConnectInput, create: MovieRelationInput, delete: MovieDeleteInput, disconnect: MovieDisconnectInput, update: MovieUpdateInput, where: MovieWhere): UpdateMoviesMutationResponse!
+                }
+
+                \\"\\"\\"Pagination information (Relay)\\"\\"\\"
+                type PageInfo {
+                  endCursor: String
+                  hasNextPage: Boolean!
+                  hasPreviousPage: Boolean!
+                  startCursor: String
+                }
+
+                type Query {
+                  actors(options: ActorOptions, where: ActorWhere): [Actor!]!
+                  actorsAggregate(where: ActorWhere): ActorAggregateSelection!
+                  actorsConnection(after: String, first: Int, sort: [ActorSort], where: ActorWhere): ActorsConnection!
+                  movies(options: MovieOptions, where: MovieWhere): [Movie!]!
+                  moviesAggregate(where: MovieWhere): MovieAggregateSelection!
+                  moviesConnection(after: String, first: Int, sort: [MovieSort], where: MovieWhere): MoviesConnection!
+                }
+
+                enum SortDirection {
+                  \\"\\"\\"Sort by field values in ascending order.\\"\\"\\"
+                  ASC
+                  \\"\\"\\"Sort by field values in descending order.\\"\\"\\"
+                  DESC
+                }
+
+                type StringAggregateSelectionNonNullable {
+                  longest: String!
+                  shortest: String!
+                }
+
+                type StringAggregateSelectionNullable {
+                  longest: String
+                  shortest: String
+                }
+
+                type UpdateActorsMutationResponse {
+                  actors: [Actor!]!
+                  info: UpdateInfo!
+                }
+
+                type UpdateInfo {
+                  bookmark: String
+                  nodesCreated: Int!
+                  nodesDeleted: Int!
+                  relationshipsCreated: Int!
+                  relationshipsDeleted: Int!
+                }
+
+                type UpdateMoviesMutationResponse {
+                  info: UpdateInfo!
+                  movies: [Movie!]!
+                }"
+            `);
+        });
+
+        describe("on INTERFACE", () => {
+            test("aggregate argument set as false", async () => {
```

Comment:
Would you not call this a no-op too as whatever this argument is set to it doesn't do anything?

```suggestion
            test("aggregate argument set as false, (no-op as abstract does not support aggregation)", async () => {
```